CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.4.5
+
+- added support of variable length arguments longer than 128 characters
+
```

Comment:
can you add this change under 4.4.4, we didn't release that one yet

Reply:
ok